minecraft/src/main/java/net/fabricmc/loader/impl/game/minecraft/MinecraftGameProvider.java

```diff
@@ -318,10 +320,22 @@ public void initialize(FabricLauncher launcher) {
 				obfJars.put("realms", realmsJar);
 			}
 
+			String sourceNamespace = "official";
+
+			try {
+				VersionPredicate mergedVersionPredicate = VersionPredicate.parse(">=1.3");
+
+				if (!mergedVersionPredicate.test(Version.parse(getNormalizedGameVersion()))) {
```

Comment:
Could this be a system property to enable (or disable?)? You should still be able to easily set when installing loader.

Reply:
Do you mean a system property for "merged intermediaries" or for the source namespace?

Reply:
I think its important for there to still be a way to still be able to use the previous "merged" intermediaries on these older versions. Another option could be to just check what source namespaces are present in the current mapping set.

Reply:
what should be the default then? previous or new "merged"?